repository/src/main/resources/db/migration/V20220314_13136__RDM-13136-add_new_subfields_to_LinkReason.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO public.field_type (created_at, reference, base_field_type_id, collection_field_type_id, version)
+VALUES (now(), 'ReasonForLinkList',
+        (select id from field_type where reference = 'Collection' and version = 1 and jurisdiction_id is null),
+        (select id from field_type where reference = 'LinkReason' and version = 1 and jurisdiction_id is null), '1');
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
```

Comment:
the other commands refer to `public.complex_field` - no biggie, just consistency

Reply:
Done in [d6abb84](https://github.com/hmcts/ccd-definition-store-api/pull/1180/commits/d6abb8451c9765671654a5e646b9d6a6cd493184)